Node JS/session fswd 01/UI/food-adda/src/components/Signup.jsx

```diff
@@ -0,0 +1,149 @@
+          <AiOutlineCloseCircle
+            size={24}
+            style={{
+              position: "absolute",
+              top: "10px",
+              right: "10px",
+              cursor: "pointer",
+            }}
+            onClick={props.onHide}
+          />
+          <h4>Sign Up</h4>
+          {/* Sign up form goes here */}
+          <div className="signup-container d-flex flex-column align-items-center">
+            <img
+              src="https://images.unsplash.com/photo-1599305445671-ac291c95aaa9?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8bG9nb3xlbnwwfHwwfHw%3D&auto=format&fit=crop&w=800&q=60"
+              className="w-75 rounded"
+              style={{ maxWidth: "300px" }}
+              alt=""
+            />
+            <Form className="mt-4 w-100" onSubmit={handleSubmit}>
+              {/* A react bootstrap switch to choose between Chef or Customer */}
+              <div className="d-flex justify-content-between mb-3">
+                <Form.Label className="h5">Register as:</Form.Label>
+                <div>
+                  <Form.Check
+                    inline
+                    label="Chef"
+                    name="group1"
+                    value={"chef"}
+                    type="radio"
+                    id="radio-1"
+                    required
+                  />
+                  <Form.Check
+                    inline
+                    label="Customer"
+                    name="group1"
+                    value={"customer"}
+                    type="radio"
+                    id="radio-2"
+                  />
+                </div>
+              </div>
+              <Form.Group className="mb-3" controlId="registerMail">
+                <Form.Control type="email" placeholder="Enter email" />
+              </Form.Group>
+
+              {/* Password not matching alert */}
+              {pswd !== confirmPswd && confirmPswd !== "" ? (
+                <div className="alert alert-danger" role="alert">
+                  Passwords do not match!
+                </div>
+              ) : null}
+
+              <Form.Group className="mb-3" controlId="registerPassword">
+                <Form.Control
+                  type="password"
+                  placeholder="Password (min 6 characters)"
+                  onChange={(e) => {
+                    setPswd(e.target.value);
+                  }}
+                />
+              </Form.Group>
+
+              <Form.Group className="mb-3" controlId="registerConfirmPassword">
+                <Form.Control
+                  type="password"
+                  placeholder="Confirm Password"
+                  onChange={(e) => {
+                    setConfirmPswd(e.target.value);
+                  }}
+                />
+              </Form.Group>
+              <Button
+                type="submit"
+                className="bg-dark w-100"
+                disabled={pswd.length < 6 || confirmPswd.length < 6}
```

Comment:
We should give a message to the user under the password text, as they might get confused. As to why the signup text is disabled, when he has typed password with less than 6 characters